### DISCUSSION TITLE
Explain color choice. v2

### DISCUSSION THREAD

**Discussion by weepy** [picocss/pico#377] [General]
Hi - I'd love to understand the thinking/design decisions behind the colour palettes. It's obviously similar to Material Color, but also quite different in that you have darker colours and some other bits.

**Comment by lucaslarroche** [comment]
See here:
https://v2.picocss.com/docs/v2

More contrast was needed to follow WCAG 2.1 AAA and AA standards.
With the wider use of Pico, I wanted more neutral (less bluish) colors.

**Comment by janat08** [comment]
This site claims to follow material colors: https://aribudin.github.io/tailmater/colors.html . I like the colors a lot. Would it be possible to make that a theme?

**Comment by janat08** [comment]
I found the theme generator, and it doesn't seem to change whole lot. Have you considered adopting materials color you color generator, because I suppose that it's more full featured in terms of colors changed?

**Reply by janat08** [reply]
Unless I suppose it's the bug that isn't applying the theme.
